packages/sanity/src/core/bundles/components/dialog/BundleDetailsDialog.tsx

```diff
@@ -56,6 +59,9 @@ export function BundleDetailsDialog(props: BundleDetailsDialogProps): JSX.Elemen
         await submit(submitValue)
         if (formAction === 'create') {
           setPerspective(value._id)
+          telemetry.log(CreatedRelease)
```

Comment:
A thought here, though I'm unsure if this does or doesn't align with our current telemetry strategies elsewhere. But interested in your thoughts - tracking how/where a release is created eg. from the global perspective menu or from within the releases plugin

Reply:
That's true that might be interesting to know 💡 
I'll add it! Thank you for the idea Jordan, great catch!

Reply:
Added it on https://github.com/sanity-io/sanity/pull/7460/commits/64bf480de2bd829ba86b9e3f2b5b0f8cf6fd402e

I kept the terms as `origin: 'structure' | 'release-plugin' ` because we know that the section where we create releases from the structure tool will likely change (and might not end up in the nav bar as is now) and kept the name  simple as `release-plugin` because I don't think we'll have multiple places where we'll be able to create the release within the plugin (nor do I know if we need that sort of granularity) + with the re-design it might not be in the "overview" :)

Thoughts are welcome!